bin/cli-helpers.js

```diff
@@ -1,55 +1,55 @@
 var path = require("path");
 var fsExtra = require("fs-extra");
+const { globSync } = require("glob");
 
 function loadVerifyExamplesConfig(configPath) {
   /* load the doc test config if we can find it
      otherwise, copy the template one and load that
   */
 
   var verifyExamples = null;
+  var elmJson = null;
 
   try {
     verifyExamples = require(configPath);
+    if (verifyExamples["root"] !== undefined) {
+      throw new Error(
+        "elm-verify-examples.json: 'root' is no longer a valid key. It defaults to point one directory up from `/tests`."
```

Comment:
Maybe I misunderstand, but I think this error will just be swallowed by the catch block?

Reply:
you are totally correct.

Reply:
I think it would be OK to skip it. Migration is easy enough.

Otherwise, just inform of superfluous config via `console.warn`? The tool is most likely going to work if a `root` is given, but it's a nice courtesy to inform that the config is (1) being ignored, (2) no longer needed.